lnrpc/lightning.proto

```diff
@@ -4270,11 +4282,11 @@ message RPCMessage {
 
 message RPCMiddlewareResponse {
     /*
-    The unique ID of the intercepted request that this response refers to. Must
-    always be set when giving feedback to an intercept but is ignored for the
-    initial registration message.
+    The request message ID this response refers to. Must always be set when
+    giving feedback to an intercept but is ignored for the initial registration
+    message.
     */
-    uint64 request_id = 1;
+    uint64 ref_msg_id = 1;
```

Comment:
This fix should make it into `0.14` (hopefully). So since we haven't released a version under the old name, this shouldn't really break anything yet. Except perhaps for the BOS scripts which discovered this problem in the first place.

Reply:
Yeah I think it's fine given we haven't "officially" released this API.